CHANGELOG.md

```diff
@@ -1,21 +1,40 @@
 # Changelog
 
-# 0.7.0 (UNRELEASED)
+# 0.7.0 (2019-03-12)
 
 ## Migration Disclaimer
 
 Version 0.7.0 of M3 includes a redesign of the snapshotting and commitlog components (#1384 among others). This redesign was critical to improve M3DB's consistency guarantees, reduce the amount of disk space that is wasted by commitlogs, and enable future feature development to support writing data at arbitrary times.
 
 The redesign is **backwards compatible** but not **forwards compatible**. This means that you should be able upgrade your < 0.7.0 clusters to 0.7.0 with no issues, but you will not be able to downgrade without taking some additional steps. Note that the first bootstrap after the upgrade may take longer than usual, but subsequent bootstraps should be just as fast as they used to be, or even faster.
 
-### Troubleshotting and Rolling Back
+### Troubleshooting and Rolling Back
 
 If you run into any issues with the upgrade or need to downgrade to a previous version for any reason, follow these steps:
 
 1. Stop the node that is having trouble with the upgrade or that you're trying to downgrade.
 2. Modify the `bootstrappers` config in the M3DB YAML file from `filesystem, commitlog, peers, uninitialized_topology` to `filesystem, peers, commitlog, uninitialized_topology`. This will force the node to bootstrap from its peers instead of the local snapshot and commitlog files it has on disk, bypassing any issues related to file incompatibility between versions.
 3. Turn the node back on and wait for it to finish bootstrapping and snapshotting. Once everything looks stable, change the config back to `filesystem, commitlog, peers, uninitialized_topology` so  that the next time the node is restarted it will default to using the snapshot and commitlog files.
 
+## New Features
+
+- **M3DB**: Restructuring of commitlog and snapshotting feature as described above (#1384)
+- **M3DB**: Obtain a lock on data directory on startup (#1376)
+- **M3Coordinator**: Add support for zone / environment override headers in namespace APIs so multiple M3DB clusters can be administered from the same M3Coordinator instance / etcd cluster (#1427)
```

Comment:
So very nit, but I think we should lowercase `m3coordinator`, `m3query`, `m3nsch` since they're names, whereas `M3DB` is an abbreviation

Reply:
Also, on `# 0.4.8 (2018-10-20)` and before, they just say `DB`, `Coordinator`, etc. so we should definitely be consistent

Reply:
To the latter point, I think it's fine that your changelogs become a bit more polished and expressive as time and adoption goes up; probably no reason to be shackled to conventions which were largely ad-hoc decisions